_posts/2020-05-11-bye-bye-pulp2.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Katello - Saying goodbye to Pulp 2
+date: 2020-05-11 14:19:53
+author: Justin Sherrill
+tags:
+- katello
+- pulp
+---
+
+<!--more-->
+
+Hello All!
+
+With the release of Katello 3.15 we have introduced Pulp 3 as a backend service to manage content.  Over the next couple of releases we will phase in Pulp 3 and phase out Pulp 2:
+
+* 3.15:  Introduction of Pulp 3, File and Docker content can use Pulp 3
+* 3.16:  Yum and Debian content can use Pulp 3
+* 4.0:   Pulp 2 is no longer part of Katello, upgrading will remove it
+
+With 3.15 you can start [migrating content over to Pulp 3](https://theforeman.org/plugins/katello/3.15/upgrade/index.html#pulp3-migration), or you can wait until 3.16 is available.  You will need to go through the migration process before upgrading to Katello 4.0.
+
+Benefits
+========
+
+Some improvements Pulp 3 provides over Pulp 2 are:
+
+* A Simplified Architecture should lead to more content types being available over time
+* Standardization on Postgresql across all Katello components (No more MongoDB!)
```

Comment:
I'd also say that the deployment model is much easier to understand. The Apache config was very complex in Pulp 2 while Pulp 3's reverse proxy model is very easy to understand.

Reply:
added a mention for that, thanks!